tsnet/tsnet.go

```diff
@@ -178,8 +188,34 @@ func (s *Server) start() error {
 		return fmt.Errorf("%v is not a directory", s.rootPath)
 	}
 
-	// TODO(bradfitz): start logtail? don't use filch, perhaps?
-	// only upload plumbed Logf?
+	cfgPath := filepath.Join(s.rootPath, "tailscaled.log.conf")
+	collection := logtail.CollectionNode
+
+	lpc, err := logpolicy.ConfigFromFile(cfgPath)
+	if err != nil {
+		logf("logpolicy.ConfigFromFile %v: %v", cfgPath, err)
```

Comment:
why a different behavior from tailscaled?  I saw you had that, but couldn't rationalize why we would treat tsnet differently.

Reply:
Several thoughts:
1. Logging, but not erroring, is the prior behavior of `logpolicy`, so we shouldn't change that.
2. `tailscaled` prioritizes starting up at all costs.
    * For example, if you run `tailscaled` on a Raspberrry Pi in the woods, ensuring connectivity is of paramount importance. You wouldn't want to drive out to the woods to diagnose the problem.
3. `tsnet` is most often used in managed systems where the guarantee that things start up exactly as specified is more likely higher stronger prioty.
    * As a user of `tsnet`, I have a strong expectation that the logs for a given service will always go to a specific log stream ID. In the rare event of an IO error causing `LoadFromConfig` to fail, I'd rather have `tsnet` complain and for me to look into the issue than for it to automatically regenerate a new logstream ID and overwrite the original log config file.
    * I should have access to diagnose the problem since I can SSH into the machine and poke around.

Reply:
👍🏻 I can buy that.